modules/nested-accordion/widgets/nested-accordion.php

```diff
@@ -153,13 +251,36 @@ protected function register_controls() {
 		$this->end_controls_section();
 	}
 
+	protected function is_active_icon_exist($settings): bool {
```

Comment:
why protected ?

Reply:
Fair enough I guess we won't be extending that. I've changed these to private.